recipes/lyra/meta.yaml

```
@@ -0,0 +1,49 @@
+{% set name = "lyra" %}
+{% set version = "1.6.1" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://github.com/bfgroup/Lyra/archive/{{ version }}.tar.gz
+  sha256: a93f247ed89eba11ca36eb24c4f8ba7be636bf24e74aaaa8e1066e0954bec7e3
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - cmake
+    - {{ compiler('cxx') }}
+    - {{ stdlib("c") }}
+    - ninja
+
+test:
+  requires:
+    - cmake
+    - {{ compiler('cxx') }}
+    - ninja
+  files:
+    - test
+
+  commands:
+    - test -f $PREFIX/include/lyra/lyra.hpp  # [unix]
+    - if not exist %PREFIX%\\Library\\include\\lyra\\lyra.hpp exit 1  # [win]
+    - test -f $PREFIX/share/lyra/cmake/lyraConfig.cmake  # [unix]
+    - if not exist %PREFIX%\\Library\\share\\lyra\\cmake\\lyraConfig.cmake exit 1  # [win]
+    - cd test
+    - cmake . -GNinja -DCMAKE_BUILD_TYPE=Release
+    - cmake --build . --config Release
+    - ./program  # [unix]
+    - program.exe  # [win]
+
+about:
+  home: https://github.com/bfgroup/Lyra
+  license: BSL-1.0
+  license_file: LICENSE.txt
+  summary: A simple to use, composing, header only, command line arguments parser for C++ 11 and beyond
+
+extra:
+  recipe-maintainers:
+    - apmorton
```

@mattgodbolt Do you also want to be listed as a maintainer and get access on the feedstock later on?
```suggestion
    - apmorton
    - mattgodbolt
```

---

Good point! @apmorton has been tutoring me in how to prepare this so suggested he should maintain, but makes sense too for me! thanks for the suggestion.